src/Sarif/Autogenerated/AnnotatedCodeLocation.cs

```diff
@@ -47,7 +48,7 @@ public SarifNodeKind SarifNodeKind
         /// Key/value pairs that provide additional information about the code location.
         /// </summary>
         [DataMember(Name = "properties", IsRequired = false, EmitDefaultValue = false)]
-        public IDictionary<string, string> Properties { get; set; }
+        internal override IDictionary<string, SerializedPropertyInfo> Properties { get; set; }
```

Comment:
Those property access APIs in the base class need access to the actual `Properties` dictionary. So declare the dictionary `abstract` in the base class, and override it here. (When discussed this on Friday, I couldn't get it to work. Turned out I needed to add some attributes to the property in the base class to get it to serialized properly.)